src/Symfony/Component/Routing/RouteCompiler.php

```diff
@@ -252,7 +252,7 @@ private static function determineStaticPrefix(Route $route, array $tokens): stri
      */
     private static function findNextSeparator(string $pattern, bool $useUtf8): string
     {
-        if ('' == $pattern) {
+        if ('' === $pattern) {
             // return empty string if pattern is empty or false (false which can be returned by substr)
```

Comment:
the comment is invalid because $pattern is a string

Reply:
Even with the old version, it was impossible to have `false` since it's typed in function prototype 🤔